server/lib/date-adjust.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+const fs = require("fs");
+
+// Write recent dates to initial-tweets.json
+// Specifically sync to not interfere with student functions
+module.exports = () => {
+  // Path to the initial tweets
+  const tweetsPath = "./server/data-files/initial-tweets.json";
+  // Read the tweets synchronously
+  const tweetsRead = JSON.parse(fs.readFileSync(tweetsPath, { encoding: "utf8" }));
```

Comment:
Fairly certain we can just require a json file with needing to parse it or fs.read it.

Reply:
https://nodejs.org/api/modules.html#modules_require_id

You are correct!  On it.